### ISSUE TITLE
Ability to hide titlebar

### ISSUE THREAD

**Issue by zenzoa** [lapce/lapce#2368]
## Is your feature request related to a problem? Please describe.

I don't need most of the functions on the titlebar - ssh, navigation, dedicated search bar/command palette space. It takes up space and creates visual clutter that I'd rather use to focus on my code files, especially on a tiling wm which does not have titlebars in the first place. I find the blue "ssh" button especially distracting because the color makes it visually prominent.

## Describe the solution you'd like

I would love the ability to turn off the titlebar and/or customize what's shown on it. `CTRL-P` can still let me open a file from the workspace, and `CTRL-SHIFT-P` can still open the command palette, but it would slide in/pop over the editor screen. VSCode/ium has this feature and I really love it.

## Additional context

In general, I enjoy having a minimal UI. Other examples of features I'd enjoy would include:
- ability to turn off the prev/next tab buttons
- ability to turn off icons and close buttons in tabs
- ability to turn off split and close current buttons from tab bar
- ability to toggle visibility of status bar
- ability to toggle visibility of sidebar(s)
- ability to turn off sidebar tabs when only one tab is in the sidebar
- ability to turn off Open Editors list

Thanks so much for making such a great, speedy editor, I am looking forward to using it more and seeing how it develops!

**Comment by joaotolovi** [comment]
> I find the blue "ssh" button especially distracting because the color makes it visually prominent.

I agree I see no reason to highlight the button

**Comment by jbr** [comment]
Came here to file this exact issue. I have no need for remote editing and likely never will. The bright blue button seems like it's calling out attention like a notification, and I would like to dismiss it

**Comment by Wcubed** [comment]
I agree with the remote button being overly attention grabbing. Luckily, that specific color is editable in the theme settings.
![afbeelding](https://github.com/lapce/lapce/assets/7442115/a8e11fe9-1677-4749-bf51-11cdc7cb7cc9)

**Comment by smir-ant** [comment]
up

**Comment by Gaiko-sw** [comment]
If the top bar was gone, you could hit tabs and sidebar buttons by jamming the mouse into the top pixel of the screen.

You should currently be able to get the hamburger menu, back and forward buttons, file picker, start debugging, and settings button, but for some reason only the remote session button touches the top pixel. 

Besides debug, I don't think I need the other items often, or would be more likely to use a keyboard shortcut. 

The option 'Custom Titlebar' looked like it would toggle the lapce top bar, but all it does is disable the window manager's bar and puts window buttons in lapce's. That seems a lot more like a 'Disable Native Titlebar'.
